website/docs/reference/notifications.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Notifications
+---
+
+:::info Availability
+
+Notifications were introduced in Unleash 4.22.0 for pro and enteprise customers.
+
+:::
+
+Notifications give your project members hints about interesting events happening in their projects.
+To see your notifications, click on the ring bell icon in the top-right corner of the screen.
+You will only see the notifications from the other project members without the events triggered by yourself.
```

Comment:
I wonder if rewriting it to something like this works better?

Also, is it possible to disable notifications entirely?

```suggestion
Unleash's notifications give you updates when certain events occur in projects that you are part of. The notifications are accessible from the notifications button in the navigation section of the admin UI.

Your notifications will only contain updates from other members; you will not get notifications about actions you perform yourself.
```

For future reference: describing the visual appearance of a button (or where on the page it is) has no real value to users with visual impairments. I also checked the markup of it, and the button doesn't seem to have any text associated with it. So it's a side task, but we should try and make sure that the button has some text that screen readers can latch onto before this goes out (or as soon as possible).

Reply:
Good catch with being overly specific with the UI details